src/layouts/Home/components/Search.tsx

```diff
@@ -9,13 +9,12 @@ import React from 'react';
 import { useStaticQuery, graphql } from 'gatsby';
 import Img from 'gatsby-image';
 import { css, ThemeProps, DefaultTheme } from 'styled-components';
-import { getLineHeight } from '@zendeskgarden/react-theming';
+import { getLineHeight, mediaQuery } from '@zendeskgarden/react-theming';
 import { Grid, Row, Col } from '@zendeskgarden/react-grid';
 // import { MediaInput } from '@zendeskgarden/react-forms';
```

Comment:
[nit] can we remove these commented imports?

Reply:
I think they go together with commented-out search code below. But, yeah, never a fan of commented code.